aspnetcore/blazor/components.md

```diff
@@ -238,22 +238,22 @@ If the value of the `ParentYear` property is changed by selecting the button in
 
 The `Year` parameter is bindable because it has a companion `YearChanged` event that matches the type of the `Year` parameter.
 
-By convention, `<ChildComponent bind-Year="@ParentYear" />` is essentially equivalent to writing,
+By convention, `<ChildComponent @bind-Year="@ParentYear" />` is essentially equivalent to writing,
 
 ```cshtml
-<ChildComponent bind-Year-YearChanged="@ParentYear" />
+<ChildComponent @bind-Year="@ParentYear" @bind-Year:event="YearChanged" />
```

Comment:
@ajaybhargavb - do we have a test for what happens if you put in something other than static text in `@bind-Year:event`?

Reply:
I don't think so. I can add one though.